js/app.js

```diff
@@ -0,0 +1,234 @@
+console.log("works");
+//******************ANSWER THE FOLLOWING******************
+//1 a light switch with on/off
+//  I would store it in an object function by using lightSwitch as a key
+// and then function(){// turn on //turn off}
+
+//2 a user's email address
+//  I would store in an object called user with a key of email
```

Comment:
ok -- but string is what we were looking for (look up K.I.S.S. in the article linked in the assignment)